### DISCUSSION TITLE
model portfolios

### DISCUSSION THREAD

**Discussion by microprediction** [shankarpandala/lazypredict#370] [Ideas]
Here's an example of using lazypredict and precise to create a portfolio of models. 

See [colab notebook](https://github.com/microprediction/precise/blob/main/examples_colab_notebooks/lazypredict_model_portfolio.ipynb)

There are lots of possibilities for creating "EvenLazierPredict", it would seem :)

**Comment by BradKML** [comment]
Of course since these things exist: https://github.com/automl and https://pycaret.gitbook.io/docs/ and https://github.com/mljar/mljar-supervised
